code/ui/blocks/src/components/ArgsTable/types.ts

```diff
@@ -41,6 +41,10 @@ export interface ArgType {
   description?: string;
   defaultValue?: any;
   if?: Conditional;
+  table?: {
+    readonly?: boolean;
```

Comment:
If the table type was missing, should you then add the other missing properties?
```category, defaultValue, disable, subcategory, type```

Reply:
I definitely can improve the typing here!